manifests/cmp-sidecar/argocd-repo-server.yaml

```diff
@@ -24,28 +25,13 @@ spec:
         image: registry.access.redhat.com/ubi8
         env:
           - name: AVP_VERSION
-            value: 1.11.0
```

Comment:
We'll need to keep this line so that `AVP_VERSION` is defined

Reply:
Oops definitely not intended!